Assets/AltTester/Runtime/UI/AltDialog.cs

```diff
@@ -334,6 +338,8 @@ private void setUpAppNameInputField()
         private void onAppNameValueChanged(string _)
         {
             setInteractibilityForRestartButton(true);
+            string message = $"Editing host, port or appName.{Environment.NewLine}Press the Restart button to start connection with the new values.";
```

Comment:
Create this message as a const, because everything is static. And use that variable in every method